html/js/profile.js

```diff
@@ -66,8 +66,8 @@ profile.setProfileValue = function () {
         profile.editProfileImage();
     });
     let aImg = cm.createQRCodeImg('https://chart.googleapis.com/chart?cht=qr&chs=177x177&chl=' + cellUrl);
-    $(".user-qr-code").html($(aImg));
-    $(".user-cell-url").html(cm.getMyCellUrl());
+    $(".user-qr-code").text($(aImg));
```

Comment:
Can you verify if the image tag is proper rendered? If not, we should use html().

Reply:
![image](https://user-images.githubusercontent.com/29846750/51585793-1ac35680-1f1e-11e9-993b-2eff3fae68cf.png)